rss2email/feeds.py

```diff
@@ -242,37 +265,19 @@ def load(self, lock=True, require=False):
 
     def _load_feeds(self, lock, require):
```

Comment:
I would guess the `lock` argument is no longer required

Reply:
Yeah. And actually, it was always called with lock=True anway so it was never required.